packages/ember-input-validation/src/components/prefabs/tpk-validation-integer.gts

```diff
@@ -16,14 +16,20 @@ export interface TpkValidationIntegerComponentSignature
     | 'step'
     | 'unmaskValue'
     | 'maskOptions'
-  >;
+  > & {
+    unsignedInteger?: boolean;
+  };
   Blocks: {
     default: [];
   };
   Element: HTMLDivElement;
 }
 
 export default class TpkValidationIntegerComponent extends BaseValidationComponent<TpkValidationIntegerComponentSignature> {
+
+get unsignedInteger() {
+  return this.args.unsignedInteger? 0 : undefined;
```

Comment:
return this.args.unsignedInteger ?? 0;

Reply:
c'est une bolean true = 0 et false = undefined

Reply:
c'est pas unsigned c'est minimum alors